demos/build.gradle

```diff
@@ -1,26 +1,18 @@
 plugins {
-  id 'org.akhikhl.gretty' version '1.4.0'
+  id 'org.akhikhl.gretty' version '2.0.0'
 }
 
 subprojects {
-  configurations.all {
-    resolutionStrategy {
-      // It seems jetty has some internal conflict and so those need to be forced
-      force 'org.ow2.asm:asm:5.0.3', 'org.ow2.asm:asm-commons:5.0.3', 'org.glassfish:javax.el:3.0.1-b08'
-    }
-  }
-
   apply plugin: 'war'
   apply plugin: 'org.akhikhl.gretty'
 
   gretty {
-    port = 8080
```

Comment:
Is this now the default?  Was it not before?  Whats going on here?

Reply:
It think it was indeed the default before. And it is now. But the port variable has became I don't know what so removing it was my best move.